main.go

```diff
@@ -20,6 +20,7 @@ type Params struct {
 	Password      *string  `json:"password,omitempty"`
 	Repository    *string  `json:"repository,omitempty"`
 	Username      *string  `json:"username,omitempty"`
+	Tag      	  *string  `json:"tag,omitempty"`
```

Comment:
This line appears to be formatted incorrectly. Can you run `go fmt` on your changes?